### PR TITLE
Fix docs

### PR DESCRIPTION
**Pull request by jiminhsieh**

### CODE REVIEW

File: akka-docs/src/main/paradox/general/actor-systems.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ the manager knows which kind of failures are expected and how to handle
 them.
  * If one actor carries very important data (i.e. its state shall not be lost
 if avoidable), this actor should source out any possibly dangerous sub-tasks
-to children it supervises and handle failures of these children as
+to children it supervises and handles failures of these children as
```
Comment (raboof) [inline akka-docs/src/main/paradox/general/actor-systems.md]: 'this actor should source out (..) and handle failures (..)' was actually correct, right?

File: akka-docs/src/main/paradox/general/message-delivery-reliability.md [hunk 2]
```diff
@@ -47,7 +47,7 @@ When it comes to describing the semantics of a delivery mechanism, there are
 three basic categories:
 
  * **at-most-once** delivery means that for each message handed to the
-mechanism, that message is delivered zero or one times; in more casual terms
+mechanism, that message is delivered zero or one time; in more casual terms
```
Comment (raboof) [inline akka-docs/src/main/paradox/general/message-delivery-reliability.md]: Tricky: google suggests "zero or one times" is actually more common than "zero or one time". However perhaps we can find an even better way to rewrite this sentence?

File: akka-docs/src/main/paradox/general/message-delivery-reliability.md [hunk 3]
```diff
@@ -177,8 +177,8 @@ particular:
 >
 > Parent actor `P` might receive the two events either in order `M`, `F` or `F`, `M`
 
-The reason for this is that internal system messages has their own mailboxes therefore the ordering of enqueue calls of
-a user and system message cannot guarantee the ordering of their dequeue times.
+The reason for this is that internal system messages have their own mailboxes therefore the ordering of enqueue calls of
+a user and system messages cannot guarantee the ordering of their dequeue times.
```
Comment (raboof) [inline akka-docs/src/main/paradox/general/message-delivery-reliability.md]: 'a' needs to be dropped here as well, I think, right?

**Review by raboof — COMMENTED** [review]
Thanks for looking at those, I added a couple of comments!

**Comment by raboof** [comment]
OK TO TEST

File: akka-docs/src/main/paradox/general/message-delivery-reliability.md (hunk 2 above)
Comment (jiminhsieh) [reply akka-docs/src/main/paradox/general/message-delivery-reliability.md]: > google suggests "zero or one times" is actually more common than "zero or one time".

I didn't notice before. 

But I find [this](https://docs.microsoft.com/en-us/dotnet/standard/base-types/quantifiers-in-regular-expressions) at MS document
> Match zero or one time.

> we can find an even better way to rewrite this sentence?

How about **none or once**?

File: akka-docs/src/main/paradox/general/message-delivery-reliability.md (hunk 3 above)
Comment (jiminhsieh) [reply akka-docs/src/main/paradox/general/message-delivery-reliability.md]: If `a` is dropped, it would be changed to `users`. I think both are correct.

Comment (raboof) [reply akka-docs/src/main/paradox/general/message-delivery-reliability.md]: I don't think that's accurate here, as "user and system messages" here is short for "user messages and system messages".

Perhaps this also means we should simplify this sentence. Perhaps we should split it out into multiple sentences?

File: akka-docs/src/main/paradox/general/message-delivery-reliability.md (hunk 2 above)
Comment (jiminhsieh) [reply akka-docs/src/main/paradox/general/message-delivery-reliability.md]: Yeah, I will use this one - `that message is delivered once or not at all`.

File: akka-docs/src/main/paradox/general/message-delivery-reliability.md (hunk 3 above)
Comment (jiminhsieh) [reply akka-docs/src/main/paradox/general/message-delivery-reliability.md]: Oh, thank for the correction! How about just a little verbose for the document? I use `user messages and system messages`.

**Review by patriknw — APPROVED** [review]
LGTM, nice

**Review by raboof — APPROVED** [review]

**Pull request merged**